content/es/docs/concepts/architecture/master-node-communication.md

```diff
@@ -0,0 +1,59 @@
+---
+reviewers: 
+- glo-pena
+title: Comunicación Nodo-Maestro
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Este documento cataloga las diferentes vías de comunicación entre el nodo máster (en realidad el apiserver) y el clúster de Kubernetes. La intención es permitir a los usuarios personalizar sus instalaciones para proteger sus configuraciones de red de forma que el clúster pueda ejecutarse en una red de no confianza. (o en un proveedor de servicios en la nube con direcciones IP públicas)
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+### Clúster a Máster
+ 
+Todos los canales de comunicación desde el clúster hacia el máster terminan en el apiserver (ningún otro componente del máster está diseñado para exponer servicios remotos). En un despliegue típico, el apiserver está configurado para escuchar conexiones remotas en un canal seguro cómo HTTPS en el puerto (443) con una o más formas de [autenticación de clientes](/docs/reference/access-authn-authz/authentication/) habilitada. Una o más formas de [autorización](/docs/reference/access-authn-authz/authorization/) deberían ser habilitadas, especialmente si se permiten [solicitudes anónimas](/docs/reference/access-authn-authz/authentication/#anonymous-requests)
+o [tokens de cuenta de servicio](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+
+Los nodos deben ser aprovisionados con el certificado raíz público del clúster de forma que puedan conectar de forma segura con el apiserver en conjunción con credenciales de cliente válidas. Por ejemplo, en un despliegue por defecto en GKE, las credenciales de cliente proporcionadas al kubelet están en la forma de certificados de cliente. Accede [kubelet TLS bootstrapping](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/) para ver cómo aprovisionar certificados de cliente de forma automática.
+
+Pods que deseen conectar con el apiserver pueden hacerlo de forma segura a través de una cuenta de servicio, de esta forma Kubernetes inserta de forma automática el certificado raíz público y un bearer token válido en el pod cuando este es instanciado. El servicio `kubernetes` (en todos los namespaces) se configura con una dirección IP virtual que es redireccionada (via kube-proxy) al punto de acceso HTTPS en el apiserver.
+
+Los componentes máster también se comunican con el apiserver del clúster a través de un puerto seguro.
+
+Como resultado, el modo de operación por defecto para conexiones desde el clúster (nodos y pods ejecutándose en nodos) al máster es seguro por defecto y puede correr en redes públicas y/o de no confianza.
+
+## Máster a Clúster
+
+Hay dos vías de comunicación primaria desde el máster (apiserver) al clúster. La primera es desde el apiserver al proceso kubelet que se ejecuta en cada nodo del clúster. La segunda es desde el apiserver a cualquier nodo, pod, o servicio a través de la funcionalidad proxy del apiserver.
+
+### apiserver to kubelet
```

Comment:
s/to/al ?

Reply:
Entiendo que se refiere en que el título está inglés.

> apiserver **to** kubelet

Reply:
Tiene más sentido :D  cambiado